playbooks/common/openshift-cluster/node_docker_ca.yml

```diff
@@ -0,0 +1,128 @@
+---
+- name: Configure CA certificate for secure registry
```

Comment:
I'd like to see this logic pushed down into a role, rather than a playbook. Ideally, we'd be able to apply this as part of the node/docker config.

@abutcher maybe we need to deploy the registry prior to configuring the nodes, that would provide the needed info for node configuration.